lib/mayaUsd/resources/scripts/mayaUsdCacheMayaReference.py

```diff
@@ -34,6 +34,9 @@
 # Cache options in string format, for MEL mayaUsdTranslatorExport() consumption.
 _cacheExportOptions = None
 
+# The options string that we pass to mayaUsdTranslatorExport.
+kTranslatorExportOptions = 'all;!output-parentscope;!animation-data'
```

Comment:
Created constant for this string (used in 2 places) and added new string "output-parentscope"